config/rbac/service_account.yaml

```diff
@@ -3,3 +3,16 @@ kind: ServiceAccount
 metadata:
   name: controller-manager
   namespace: system
+---
+# As of Kubernetes 1.24, ServiceAccount tokens are no longer automatically
+# generated. Instead, manually create the secret and the token key in the
+# data field will be automatically set.
```

Comment:
I'm using `kindest/node:v1.25.2`, and I'm not getting the secret auto-generated:
```
$ kubectl get secrets,sa -n dws-operator-system
NAME                                     TYPE                                  DATA   AGE
secret/dws-operator-controller-manager   kubernetes.io/service-account-token   3      108s
secret/webhook-server-cert               kubernetes.io/tls                     3      107s

NAME                                             SECRETS   AGE
serviceaccount/default                           0         108s
serviceaccount/dws-operator-controller-manager   0         108s
```